### PR TITLE
CATROID-1339 "Add to backpack" option missing for "Define" bricks

### PR DESCRIPTION
**Pull request by friendo97**
Definitionbricks for Userdefined Bricks now have the option "add to backpack" in their context menu

https://jira.catrob.at/browse/CATROID-1339

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickContextMenuTest.kt [hunk 1]
```diff
@@ -153,4 +153,13 @@ class BrickContextMenuTest(
     fun testExpectedHelp() {
         assertTrue(contextMenuItems.contains(R.string.brick_context_dialog_help))
     }
+
+    @Test
+    fun testExpectedAddToBackpack() {
+        val showsAddToBackpack: Boolean = when (brick) {
+            is UserDefinedReceiverBrick, is WhenStartedBrick -> contextMenuItems.contains(R.string.backpack_add)
```
Comment (bengaal) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickContextMenuTest.kt:160]: I think we could add the other Event blocks here to increase the scope of the test.

**Review by bengaal — CHANGES_REQUESTED** [review]

**Comment by bengaal** [comment]
Please consider my suggestion above, otherwise the PR looks good to me.

File: catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickContextMenuTest.kt (hunk 1 above)
Comment (friendo97) [reply catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickContextMenuTest.kt]: added the other when bricks to the test

**Review by bengaal — APPROVED** [review]
Requested changes were implemented, looks good to me!

**Comment by dandrawes** [comment]
Thank you!

**Pull request merged**